From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: mozilla add branch64
Date: Tue, 03 May 2016 17:42:54 +0200
References:
https://bugzilla.mozilla.org/show_bug.cgi?id=1266366

mozilla add branch64 in MacroAssembler-none.h
already upstream as tracked by
https://www.mail-archive.com/freebsd-gecko@freebsd.org/msg05462.html

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 js/src/jit/none/MacroAssembler-none.h |    2 ++
 1 file changed, 2 insertions(+)

Index: mozilla/js/src/jit/none/MacroAssembler-none.h
===================================================================
--- mozilla.orig/js/src/jit/none/MacroAssembler-none.h
+++ mozilla/js/src/jit/none/MacroAssembler-none.h
@@ -255,6 +255,8 @@ class MacroAssemblerNone : public Assemb
     template <typename T, typename S> void branchPrivatePtr(Condition, T, S, Label*) { MOZ_CRASH(); }
     template <typename T, typename S> void decBranchPtr(Condition, T, S, Label*) { MOZ_CRASH(); }
     template <typename T, typename S> void branchTest64(Condition, T, T, S, Label*) { MOZ_CRASH(); }
+    template <typename T, typename S> void branch64(Condition, T, S, Label*) { MOZ_CRASH(); }
+    template <typename T, typename S> void branch64(Condition, T, T, S, Label*) { MOZ_CRASH(); }
     template <typename T, typename S> void mov(T, S) { MOZ_CRASH(); }
     template <typename T, typename S> void movq(T, S) { MOZ_CRASH(); }
     template <typename T, typename S> void movePtr(T, S) { MOZ_CRASH(); }
